plugins/SSH.koplugin/main.lua

```diff
@@ -95,6 +95,10 @@ function SSH:stop()
         timeout = 2,
     })
 
+    if self:isRunning() then
+        os.execute("rm /tmp/dropbear_koreader.pid")
```

Comment:
Is there a particular reason to avoid `os.remove()`?

Reply:
Oh, duh', good point ;p.

Reply:
`os.execute` will fork a shell (it's literally a thin wrapper around `system`), while `os.remove` will call... `remove` (the libc function) ;p.

So, yeah, it's much more sensible ;).

Reply:
Thanks! I've changed it accordingly.